maps/sccv_horizon/sccv_horizon.dmm

```diff
@@ -98026,11 +98036,12 @@
 	icon = 'icons/obj/items.dmi';
 	icon_state = "blueprints";
 	name = "station blueprints";
-	pixel_y = -8
+	pixel_y = -3;
+	pixel_x = 0
```

Comment:
Unneeded

Reply:
Resolved. Lowkey curious what caused that

Reply:
If you change it, basically; this would be known as a "dirty var" (the var set in the map is the same the object would have already) and we have some, eventually we'll lint for it, one day...